library/Opus/Doi/DoiManager.php

```diff
@@ -78,28 +80,14 @@ public function __construct()
     /**
      * Creates logger for DOI messages.
      *
-     * This code might be executed as part of a script or as part of a web request. Therefore it is possible that the
-     * log file is created with different permissions. The 'chmod' in the function tries to alleviate the problem by
-     * making sure that the owner and the group (usually 'wwwdata' or apache2 in general) have read/write permissions.
-     *
-     * TODO centralize code for creating log files (like a logging service that can be used by all modules)
+     * @return Zend_Log
      */
     public function getDoiLogger()
     {
         if (is_null($this->doiLog)) {
-            $logfilePath = $this->config->workspacePath . DIRECTORY_SEPARATOR . 'log' . DIRECTORY_SEPARATOR
-                . 'opus-doi.log';
-
-            $logfile = @fopen($logfilePath, 'a', false);
-            $writer = new Zend_Log_Writer_Stream($logfile);
-
             $format = '%timestamp% %priorityName%: %message%' . PHP_EOL;
-            $formatter = new Zend_Log_Formatter_Simple($format);
-            $writer->setFormatter($formatter);
-
-            $this->doiLog = new Zend_Log($writer);
-
-            chmod($logfilePath, 0660); // grant owner and group read/write permission
+            $logService = LogService::getInstance();
+            $this->doiLog = $logService->createLog('opus-doi', 'debug', $format, null);
```

Comment:
The last argument isn't necessary.

Implementing it like this means, the configuration cannot be overridden in {{application.ini}}. That is fine with me for now.